docs/SG-NEx_Bambu_tutorial.md

```diff
@@ -0,0 +1,148 @@
+# **Transcript discovery and quantification of SG-NEx samples**
+
+In this tutorial, we will perform novel transcript discovery and
+quantification on the SG-NEx samples. We will be using six Nanopore direct RNA-Sequencing
+samples, three replicates each from the A549 and
+HepG2 cell lines. The A549 cell line was extracted from lung tissues
+from a patient with lung cancer whereas HepG2 was extracted from
+hepatocellular carcinoma from a patient with liver cancer. We will use
+Bambu, a R package hosted on the Bioconductor platform to identify and
+quantify novel isoforms in these cell lines. 
+
+**Note: This tutorial may take 20 minutes to complete.**
+
+## **Content**
+
+- [Installation](#installation)
+- [Data Access and Preparation](#data-access-and-preparation) 
+- [Running Bambu](#running-bambu)
+- [Reference](#reference)
+
+## **Installation**
+
+First, we have to install Bambu. Before that, make sure you have R
+(version \>= 4.1) installed on your machine. We can install Bambu using the following command:
+
+``` bash
+R
+
+if (!requireNamespace("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+
+BiocManager::install("bambu")
+
+q()
+```
+If you want a more recent version of Bambu, you may refer to the Bambu Github repository [here](https://github.com/GoekeLab/bambu). 
+
+## **Data Access and Preparation**
+### **Download Data for Bambu**
+Next, we will need to download the required data to run Bambu. The required data include:
+
+-   a set of aligned reads to the genome from the A549 and HepG2 cell lines (bam files),
+-   reference human genome annotations (gtf file, TxDb object, or Bambu
+    Annotation object),
+-   reference human genome sequence (fasta file or BSgenome).
+
+Generally, you may want to learn how to get access to these data using the [data
+access
+tutorial](https://github.com/GoekeLab/sg-nex-data/blob/updated-documentation/docs/AWS_data_access_tutorial.md). Below we only show the necessary steps to download the required data. The following command requires you to have [AWS CLI](https://aws.amazon.com/cli/) installed.
+
+``` bash
+# create a directory to store the data
+mkdir bambu_tutorial
+
+# download genome fasta file 
+aws s3 cp --no-sign-request s3://sg-nex-data/data/annotations/genome_fasta/Homo_sapiens.GRCh38.dna_sm.primary_assembly.fa ./bambu_tutorial
```

Comment:
forgot to comment on this, have you tried to use the annotations and fasta file here to run: http://sg-nex-data.s3-website-ap-southeast-1.amazonaws.com/#data/data_tutorial/annotations/

they are subsetted to be corresponding to the downsamples samples, if use them, the running time should be very fast!

Reply:
resolved in abe99a9a856b20ea030ba7b1bdd3b460c1e8c4b6